treeherder/perf/alerts.py

```diff
@@ -247,13 +248,13 @@ def _multi_criterion_sort(self, relevant_alerts):
         return sorted_alerts
 
 
-class IdentifyRetriggerables:
-    def __init__(self, max_data_points: int, time_interval: datetime.timedelta, logger=None):
+class IdentifyRetriggerableDataPoints:
```

Comment:
This still sounds too generic. This class works on a single alert level. I suggest we rather name it to something like `IdentifyAlertRetriggerables` or `AlertRetriggerablesFinder`.

Reply:
Ok, `IdentifyAlertRetriggerables` will be used old `IdentifyRetriggerableDataPoints`.
Also, `IdentifyRetriggerableAlerts` will be renamed into `AutomaticAlertsSelector` in order to avoid very similar naming for different classes.